sklearn/datasets/lfw.py

```diff
@@ -155,7 +155,14 @@ def _load_imgs(file_paths, slice_, color, resize):
     for i, file_path in enumerate(file_paths):
         if i % 1000 == 0:
             logger.info("Loading face #%05d / %05d", i + 1, n_faces)
-        face = np.asarray(imread(file_path)[slice_], dtype=np.float32)
+
+        # Checks if jpeg reading worked. Refer to issue #3594 for more
+        # details.
+        img = imread(file_path)
+        if img.ndim is 0:
+            raise ValueError("Please make sure that libjpeg is installed")
```

Comment:
I don't really think this is a `ValueError`, as it does not result from input that is of the right type but wrong value. A `RuntimeError` or perhaps `EnvironmentError` might be more appropriate.